providers/hive.go

```diff
@@ -17,10 +17,7 @@ func init() {
 		},
 		//nolint:lll
 		Media: &Media{
-			DarkMode: &MediaTypeDarkMode{
```

Comment:
Seems like you accidentally deleted the DarkMode logos, can you bring them back?

Reply:
Sorry, I'll add the DarkMode Logos and upsert in the BulkWriteSupport.